erica/erica_legacy/elster_xml/grundsteuer/elster_eigentuemer.py

```diff
@@ -124,22 +120,28 @@ class EEmpfangsbevollmaechtigter:
     E7404627: Optional[str]
     E7404622: str
     E7412201: Optional[str]
-    # TODO E7412901
-
-    def __init__(self, input_data: Empfangsbevollmaechtigter):
-        self.E7404610 = elsterify_anrede(input_data.name.anrede)
-        self.E7404614 = input_data.name.titel
-        self.E7404613 = input_data.name.vorname
-        self.E7404611 = input_data.name.name
-        self.E7404624 = input_data.adresse.strasse
-        self.E7404625 = input_data.adresse.hausnummer
-        self.E7404626 = input_data.adresse.hausnummerzusatz
-        self.E7404640 = input_data.adresse.plz
-        self.E7404627 = input_data.adresse.postfach
-        self.E7404622 = input_data.adresse.ort
+    E7412901: Optional[int]
+
+    def __init__(self, eigentuemer: EigentuemerInput):
+        empfangsbevollmaechtigter = eigentuemer.empfangsbevollmaechtigter
+        self.E7404610 = elsterify_anrede(empfangsbevollmaechtigter.name.anrede)
+        self.E7404614 = empfangsbevollmaechtigter.name.titel
+        self.E7404613 = empfangsbevollmaechtigter.name.vorname
+        self.E7404611 = empfangsbevollmaechtigter.name.name
+        self.E7404624 = empfangsbevollmaechtigter.adresse.strasse
+        self.E7404625 = empfangsbevollmaechtigter.adresse.hausnummer
+        self.E7404626 = empfangsbevollmaechtigter.adresse.hausnummerzusatz
+        self.E7404640 = empfangsbevollmaechtigter.adresse.plz
+        self.E7404627 = empfangsbevollmaechtigter.adresse.postfach
+        self.E7404622 = empfangsbevollmaechtigter.adresse.ort
 
         # input_data.telefonnummer might not be set -> handle specifically
-        if hasattr(input_data, "telefonnummer") and input_data.telefonnummer:
-            self.E7412201 = input_data.telefonnummer.telefonnummer
+        if hasattr(empfangsbevollmaechtigter, "telefonnummer") and empfangsbevollmaechtigter.telefonnummer:
```

Comment:
I believe neither this nor any of the other hasattr checks are actually necessary. Pydantic is already putting explicit keys in its model with value == None for all absent fields in input, so this never evaluates to false. Also all tests passed after I removed all hasattr calls.

Once (if!) we simplify the input model for eigentuemer and gebaeude, even the second check would not be required here: you could just set the elster field to input_data.telefonnummer regardless of its value. The check would only be required for cases where there's a constructor call.